### PR TITLE
{devel,lib}[GCCcore/7.3.0,foss/2018b] TensorFlow v1.13.1, Bazel v0.20.0 (WIP)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1642~~

### CODE REVIEW

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.13.1-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        'modulename': 'keras_applications',
+    }),
+    ('Keras-Preprocessing', '1.0.9', {
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Preprocessing'],
+        'source_tmpl': 'Keras_Preprocessing-%(version)s.tar.gz',
+        'checksums': ['5e3700117981c2db762e512ed6586638124fac5842170701628088a11aeb51ac'],
+        'modulename': 'keras_preprocessing',
+    }),
+    # FIXME new dep, should be installed from source rather using .whl?
+    ('tensorflow-estimator', '1.13.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.13.1-foss-2018b-Python-3.6.6.eb]: @akesandgren Thoughts on this?

Comment (akesandgren) [reply easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.13.1-foss-2018b-Python-3.6.6.eb]: good question... also bazel based i guess?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3140.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/13f214594027450dcbfe066b3a82aee1 for a full test report.

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.13.1-foss-2018b-Python-3.6.6.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.13.1-foss-2018b-Python-3.6.6.eb]: I don't think it's worth the effort right now to try and get `tensorflow-estimator` built from source...

For now, we can take the shortcut of just installing the `.whl`, and revisit later, so I'll drop the `FIXME`.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2562.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b0610425000a93adc7fa527d6cf1adfb for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 4 out of 5 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/1c5e1085df490447d527b9deafd4257f for a full test report.

**Comment by boegel** [comment]
@akesandgren Are you using the updated `TensorFlow` easyblock from https://github.com/easybuilders/easybuild-easyblocks/pull/1642?

**Comment by akesandgren** [comment]
Probably not, didn't check the top message...

**Comment by akesandgren** [comment]
Wasn't using it. redoing the test report

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/699d3645567af163dca0b0bb1e8be88f for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2118.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/557d809b0165a0e04353e97a343ad729 for a full test report.

**Pull request merged**